presto-common/src/main/java/com/facebook/presto/common/RuntimeMetricName.java

```diff
@@ -38,6 +38,8 @@ private RuntimeMetricName()
     public static final String GET_TABLE_HANDLE_TIME_NANOS = "getTableHandleTimeNanos";
     public static final String GET_COLUMN_HANDLE_TIME_NANOS = "getColumnHandleTimeNanos";
     public static final String GET_COLUMN_METADATA_TIME_NANOS = "getColumnMetadataTimeNanos";
+    public static final String GET_HIVE_PARTITIONS_BY_NAMES_TIME_NANOS = "getHivePartitionsByNamesTimeNanos";
+    public static final String GET_HIVE_TABLE_TIME_NANOS = "getHiveTableTimeNanos";
```

Comment:
Can we just say  `GET_PARTITIONS_BY_NAMES_TIME_NANOS= "getPartitionsByNamesTimeNanos"` and then make changes in the presto-facebook code to use get**Prism**PartitionsByNamesTimeNanos instead ?

Reply:
addressed the comment thanks